backend/fms_core/migrations/0044_v3_13_0.py

```diff
@@ -0,0 +1,52 @@
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+from django.contrib.auth.models import User
+import reversion
+
+ADMIN_USERNAME = 'biobankadmin'
+
+def populate_sample_alias(apps, schema_editor):
+    """
+    For each biosample object we retrieve the first sample submitted associated with it and we update the alias
+    to be that of this first sample.
+
+    Args:
+        apps: apps class handle
+        schema_editor: ignore
+    """
+    Biosample = apps.get_model("fms_core", "Biosample")
+
+    with reversion.create_revision(manage_manually=True):
+        admin_user = User.objects.get(username=ADMIN_USERNAME)
+
+        reversion.set_comment(f"Populate biosamples' alias with the name of the first submitted sample.")
+        reversion.set_user(admin_user)
+
+        for biosample in Biosample.objects.all():
+            first_sample = biosample.derived_samples.order_by('created_at').first().samples.order_by('created_at').first()
```

Comment:
Don't rely on date for that please... it should technically be true but it really is a risky approach. Use samples without parent instead, i think it is more reliable than dates. It is likely equivalent in the current state of the DB but i don't want to have to test every single entry to make sure.